modules/update-restricted-signature-digests.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster/restricted-network-update.adoc
+
+:_content-type: PROCEDURE
+[id="update-restricted-signature-digests_{context}"]
+= Retrieving sha256 digests from the release image ConfigMap
+
+In order to update a cluster in a disconnected environment using the `oc adm upgrade` command with the `--to-image` option, you must reference the sha256 sum value that corresponds to your targeted release image.
+This sum value can be acquired by using the `oc get` command to retrieve information about release image signatures on your cluster and their corresponding digests.
```

Comment:
nit: maybe drop this line?  We aren't actually using `oc get ...` below, and I don't see a need to preview the procedure right  before the `Procedure` section anyway